cmd/postgres-reader/main.go

```diff
@@ -214,13 +214,13 @@ func newService(db *sqlx.DB, logger logger.Logger) readers.MessageRepository {
 		svc,
 		kitprometheus.NewCounterFrom(stdprometheus.CounterOpts{
 			Namespace: "postgres",
```

Comment:
Why don't you use `svcName` here? Dunno what Namespace `postgres` means, maybe its user DB or things DB

Reply:
fixed in postgres reader and writer